ppb/abc.py

```diff
@@ -39,26 +40,28 @@ def run(self):
         raise NotImplementedError
 
 
-class GameObject(object):
+class GameObjectABC(object, metaclass=abc.ABCMeta):
 
     @property
-    def rect(self) -> 'Rectangle':
+    @abc.abstractmethod
+    def position(self) -> Sequence[float]:
         raise NotImplementedError
 
-    @property
-    def image(self) -> 'Canvas':
+    @position.setter
+    @abc.abstractmethod
+    def position(self, value):
         raise NotImplementedError
 
-    @property
-    def scene(self) -> 'Scene':
+    @abc.abstractmethod
+    def __contains__(self, item):
```

Comment:
All GameObjects are containers?

Reply:
Not in the traditional sense. This one probably counts as a hack. Also probably shouldn't be in this PR.